README.md

```diff
@@ -12,70 +12,193 @@ Airport Challenge
+*Test 2* - When the length of planesAtAirport and airportCapacity are the same (1), atCapacity returns true  
+*Test 3* - When the length of planesAtAirport and airportCapacity are different, atCapacity returns false  
+*Test 4* - When atCapacity is true, planeLands returns an error message  
+*Test 5* - When atCapacity is true, planeLands doesn't change the length of planesAtAirport array   
+*Test 6* - When atCapacity is false, planeLands increases length of planesAtAirport array by 1  
+
+## User Story 4  
+As an air traffic controller  
+So I can get passengers on the way to their destination  
+I want to instruct the airport to let a plane take off and confirm that it is no longer in the airport  
+
+**Domain Model 4**  
+
+| Objects | Properties                   | Messages                          | Outputs |
+| ------- | -------------------------    | -----------------------------     | ------- |
+| Airport |planesAtAirport @Array[@Plane]| planeLands(@Plane)                | @Void   |
+|         |airportName@String            |                                   |         |
+|         |airportCapacity @Integer      | atCapacity()                      | @Boolean|
+|         |                              | planeTakesOff()                   | @String |
+|         |                              | isPlaneAtAirport(@Plane)          | @Boolean|
+| Plane   |planeID @String               |                                   |         |
+
+**Tests**  
+
+*Test 1* - Length of planesAtAirport array at the Airport decreases by 1 when planeTakesOff  
+*Test 2* - Confirm if plane is in planesAtAirport using isPlaneAtAirport returns true  
+*Test 3* - Confirm if plane is in planesAtAirport using isPlaneAtAirport returns false  
+*Test 4* - When planeTakesOff is called and isPlaneAtAirport returns true, length of planesAtAirport decreases by 1  
+
+## User Story 5  
+As an air traffic controller  
+To avoid confusion  
+I want to prevent asking the airport to let planes take-off which are not at the airport, or land a plane that's already landed  
+
+**Domain Model 5**  
+
+| Objects | Properties                    | Messages                          | Outputs |
+| ------- | ----------------------------  | ----------------------------      | ------- |
+| Airport |planesAtAirport @Array[@Plane] | planeLands(@Plane)                | @Void   |
+|         |airportName@String             |                                   |         |
+|         |airportCapacity @Integer       | atCapacity()                      | @Boolean|
+|         |                               | planeTakesOff(@Plane)             | @Void   |
+|         |                               | planeIsAtAirport(@Plane)          | @Boolean|
+| Plane   |planeID  @String               |                                   |         |
+
+**Tests**  
+
+*Test 1* - When planeTakesOff is called and isPlaneAtAirport returns False, an error message is returned  
```

Comment:
I'm not sure if this is the best way to return an error message and as in the future you may want the message to change, I could have tested by seeing is a string type was returned rather than the specific message